Tools-Override/tests.targets

```diff
@@ -130,6 +130,10 @@
          when the properties are set currently. -->
     <PropertyGroup>
       <TestCommandLine>$(TestCommandLine.Replace('{XunitTraitOptions}', '$(XunitTraitOptions)'))</TestCommandLine>
+
+      <!-- We also need to replace {CoverageFilter} with its true value, which should be evaluated correctly at this point. -->
+      <TestCommandLine>$(TestCommandLine.Replace('{CoverageFilter}', '$(CoverageFilter)'))</TestCommandLine>
```

Comment:
@ericstj This is the solution I ended up going with. We're doing the same thing for `{XunitTraitOptions}` right above this, so I felt that there was some precedence for it.